pkg/apis/policy/types.go

```diff
@@ -69,3 +69,14 @@ type PodDisruptionBudgetList struct {
 	unversioned.ListMeta `json:"metadata,omitempty"`
 	Items                []PodDisruptionBudget `json:"items"`
 }
+
+// Eviction evicts a pod from its node subject to certain policies and safety constraints.
```

Comment:
You wrote "done" in Reviewable but when I view the PR it doesn't seem to be updated. Did you push the new commit?